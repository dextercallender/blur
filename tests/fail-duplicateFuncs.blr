int sub(int x, int y) {
	int z = x-y;	
	return z;
}

int sub(int x) {
	int z = x-3;
	return z;
}

void main(int a)
{
	int a;
	a = 6;
}