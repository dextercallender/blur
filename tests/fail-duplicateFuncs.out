Fatal error: exception Failure("Duplicate function.")